### DISCUSSION TITLE
Pocketbase Demo vs v0.22

### DISCUSSION THREAD

**Discussion by slimesarecool** [pocketbase/pocketbase#5824] [Q&A] [answered]
Why does the pocketbase demo have way more customizabiltiy(editing system fields) and power(autogenerate patterns). Can someone explain to me how I can do this outside of the demo. (I'm new to pb but I love it ❤️)

**Comment by ganigeorgiev** [comment, marked as answer]
The live demo is using the v0.23.0-rc release from the `develop` branch _(you can find the used version in the Dashboard > bottom-right corner)_.

You can check all releases in the [repo's Releases](https://github.com/pocketbase/pocketbase/releases) section (the prebuilt binaries are under the collapsed "Assets" panel).

For list of changes you can also check the [CHANGELOG](https://github.com/pocketbase/pocketbase/blob/develop/CHANGELOG.md).

There are no fixed ETAs when v0.23.0 will be released (I'm still working on some minor polishing touches and on the new documentation).

**Comment by slimesarecool** [comment]
I was looking for the version I didn't know it was in the bottom right corner. 🤦 I'm using pockethost so I can't use the beta 😭
